apps/coscrad-frontend/src/components/nav-menu/__snapshots__/build-nav-menu-items.spec.tsx.snap

```diff
@@ -18,14 +18,6 @@ exports[`dynamic navigation menu when all properties are provided should return
     "label": "Notes",
     "link": "Notes",
   },
-  {
```

Comment:
What does this have to do with your branch? For small changes like this, take a separate branch and ask Justin for a PR in case it is relevant to his work.